website/docs/reference/resource-properties/constraints.md

```diff
@@ -191,6 +191,8 @@ models:
         data_type: date
 ```
 
+Note that by default (or when specified without a length), `varchar` in Redshift limits the maximum length of the values to 256 characters. This means that any string data exceeding 256 characters might get truncated OR return a "value too long for character type" error. To allow the maximum length, use `varchar(max)`. For example, `data_type: varchar(max)`.  
```

Comment:
Should use italics for emphasis rather than screaming case.
```suggestion
Note that Redshift limits the maximum length of the `varchar` values to 256 characters by default (or when specified without a length). This means that any string data exceeding 256 characters might get truncated _or_ return a "value too long for character type" error. To allow the maximum length, use `varchar(max)`. For example, `data_type: varchar(max)`.  
```